software/config/drc_environment.sh

```diff
@@ -114,6 +114,35 @@ setup_robot_computers()
   fi
 }
 
+setup_valkyrie_for_host_sites()
+{
+  OUTSIDE_IP=`curl -s ifconfig.co` # gets our outside IP
+  OUTSIDE_HOSTNAME=`nslookup $OUTSIDE_IP`
+
+  EDINBURGH='.ed.ac.uk'
+  MIT='mit.edu'
```

Comment:
Revised as on the observer workstation, still need to pull onto operator workstation once reconnected to network. Ready to merge